### PR TITLE
Basic cursor implementation rebased on new space API

### PR DESCRIPTION
**Pull request by vrogach2020**
StartAfterCursor is still in progress

### CODE REVIEW

File: src/main/java/io/tarantool/driver/exceptions/TarantoolMapperConfigurationException.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.msgpack.value.Value;
+
+/**
+ * Represents errors occurring when required mapper or converter not found.
+ *
+ * @author Vladimir Rogach
+ */
+public class TarantoolMapperConfigurationException extends TarantoolClientException {
+    public TarantoolMapperConfigurationException(String message) {
+        super(String.format("Failed to find required converter: %s", message));
```
Comment (akudiyar) [inline src/main/java/io/tarantool/driver/exceptions/TarantoolMapperConfigurationException.java:12]: Maybe it is a task for refactoring all such exceptions. TrantoolClientException is currently thrown in this case in the other places

**Review by akudiyar — CHANGES_REQUESTED** [review]

File: src/main/java/io/tarantool/driver/api/conditions/Conditions.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
     private final List<Condition> conditions = new LinkedList<>();
 
     private boolean descending;
+    //FIXME VR - write a comment why MAX_LIMIT is set as initial value ?
```
[the remark this answers is not in the document]
Comment (vrogach2020) [reply src/main/java/io/tarantool/driver/api/conditions/Conditions.java:47]: So why not use Long limit = null;  ?

File: src/main/java/io/tarantool/driver/exceptions/TarantoolMapperConfigurationException.java (hunk 1 above)
Comment (vrogach2020) [reply src/main/java/io/tarantool/driver/exceptions/TarantoolMapperConfigurationException.java:12]: I'll change to TrantoolClientException by now and think on refactoring...

File: src/main/java/io/tarantool/driver/api/space/TarantoolSpaceOperations.java [hunk 3]
```diff
@@ -92,4 +93,15 @@
      * @return space metadata
      */
     TarantoolSpaceMetadata getMetadata();
+
+    /**
+     * Perform a batch 'select' request.
```
[the remark this answers is not in the document]
Comment (vrogach2020) [reply src/main/java/io/tarantool/driver/api/space/TarantoolSpaceOperations.java:98]: Added some details.

**Review by akudiyar — APPROVED** [review]

**Pull request merged**